chain/chain/src/types.rs

```diff
@@ -30,7 +30,7 @@ use near_primitives::types::{
 };
 use near_primitives::version::{
     ProtocolVersion, MIN_GAS_PRICE_NEP_92, MIN_GAS_PRICE_NEP_92_FIX, MIN_PROTOCOL_VERSION_NEP_92,
-    MIN_PROTOCOL_VERSION_NEP_92_FIX,
+    MIN_PROTOCOL_VERSION_NEP_92_FIX, REDUCED_MIN_GAS_PRICE,
```

Comment:
we should have a NEP for this (so that we can reference it in the constant name -- the same as we did with NEP92)